.github/workflows/integration-tests.yml

```diff
@@ -6,6 +6,7 @@ env:
 
 on:
   pull_request:
+    types: [ labeled ]
```

Comment:
Why this change?

Reply:
this is a temporary change while working on this workflow, to prevent hogging resources.